File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int rem,sum,num;
        num = x;
        sum = 0;
        while(num>0){
            rem = num % 10;
            sum = (sum * 10) + rem;
            num = num / 10;
        }
        if(x<0){
            return false;
        }
        else if(x == sum){
            return true;
        }
        else{
            return false;
        }

    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        num = x
        reverse = 0
        if x < 0:
            return False
        else:
            while num > 0:
                remainder = num % 10
                reverse = (reverse * 10) + remainder
                num = num / 10
            if reverse == x:
                return True
        return False
        